src/experimental-layers/src/text-layer/font-atlas.js

```diff
@@ -54,9 +57,13 @@ export function makeFontAtlas(gl, fontFamily) {
   return {
     mapping,
     texture: new Texture2D(gl, {
-      pixels: canvas
-      // no need to specify texture filter here
-      // icon layer use the most accurate mode by default
+      pixels: canvas,
+      // padding is added only between the characters but not for borders
+      // enforce CLAMP_TO_EDGE to avoid any artifacts.
+      parameters: {
+        [GL_TEXTURE_WRAP_S]: GL_CLAMP_TO_EDGE,
+        [GL_TEXTURE_WRAP_T]: GL_CLAMP_TO_EDGE
```

Comment:
Shouldn't this be in multi-icon-layer.js?

Reply:
This file is creating the atlas and setting all the padding, given these wrap modes are needed for same functionality (making sure pixels from other characters not sampled), setting there here, along with texture creation.

Reply:
OK.